design/src/production-chains/raw-resources.md

```diff
@@ -0,0 +1,201 @@
+# Raw Resources
+
+These are the simple, fundamental materials that intermediate and end products are made of.
+Raw resources cannot be derived from another raw resources: they are "elemental" in that way.
+The total amount of each raw resource (other than water and energy) is conserved at each resource processing step.
+
+Different areas of the map will have different quantities of each resource to begin with.
+
+## Core
+
+Resources that should be added in an MVP demo.
+
+### Light
+
+**Acquisition:** It comes from the sun.
+
+**Uses:** Used by plants to capture energy, and provide calories to entire colony.
+
+**Handling properties:** Cannot be tranported. Blocked by shade.
+
+**Storage properties:** Cannot be stored.
+
+**Path to renewability:** The sun really does just keep burning.
+
+**Downsides of excess:** Heat, sun scorch of plants.
+
+### Water
+
+**Acquisition:** Gather from lakes, rivers, soil and rain.
+
+**Uses:** Essential for sustenance of all life. One of the primary transport mechanics of goods.
+
+**Handling properties:** Sinks into soil. Flows downstream. Cannot be carried directly, as it is a fluid.
+
+**Storage properties:** Evaporates over time.
+
+**Path to renewability:** Rainfall, incoming streams.
+
+**Downsides of excess:** Washes away goods. Drowns structures over time.
+
+### Dirt
```

Comment:
I do think it would make sense to have "Rich Dirt" (organic-compound heavy dirty?) and "Poor Dirt" (e.g. sand, mineral heavy?) --- i.e. things that most plants grow well in, and things that many plants don't grow well in (with of course, the outlier plants making for neat edge case options)

Renewability would then focus on not just "dirt" but producing rich dirt

Reply:
Yeah, I have a whole set of mechanics in mind here for post-MVP.

There's no "good soil" vs "bad soil": just "soil that has the properties that the thing I'm trying to grow wants".